src/doc/unstable-book/src/language-features/lang-items.md

```diff
@@ -276,6 +274,9 @@ the source code.
   - `fn_once`: `libcore/ops/function.rs`
   - `generator_state`: `libcore/ops/generator.rs`
   - `generator`: `libcore/ops/generator.rs`
+- Opting out
```

Comment:
Should we name it "Opting out"? What is it opting out of? Maybe we should call it "Escape hatch"?